Cargo.toml

```diff
@@ -21,6 +21,16 @@ opt-level = 3
 
 [profile.release]
 panic = "abort"
+opt-level = 3
+codegen-units = 1
+debug = 0
+debug-assertions = false
+# This is false by default for the release mode
+# However, false makes some assertions in Mcause firmware fail
+incremental = true
+lto = "fat"
+# Do we want to leave symbols?
+strip = "symbols"
```

Comment:
No need to strip symbols, they are already stripped when building the image (which is extracted from the ELF, without the metadata).